package.json

```diff
@@ -49,12 +49,11 @@
     "@babel/preset-react": "^7.16.7",
     "@babel/preset-typescript": "^7.16.7",
     "@babel/runtime": "^7.16.7",
-    "@enact/dev-utils": "^5.0.0-alpha.2",
+    "@enact/dev-utils": "github:enactjs/dev-utils#feature/WRO-3879",
     "@enact/template-sandstone": "^1.5.0-alpha.3",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.4",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^13.0.0",
```

Comment:
Please update the version to at least `13.1.0` since the replacement of `@testing-library/react-hooks` is included in this version.

Reply:
OK I will